### PR TITLE
Updated the README file

### PR DESCRIPTION
**Pull request by Nikhil-Ladha**
Signed-off-by: Nikhil-Ladha <nikhilladha1999@gmail.com>

#### What is the purpose of this pull request?

Added detailed process for setting up the project for both browser and android.
#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Requires change to documentation, which has been updated accordingly.

#### Fixes/Resolves issue number
#1 
#### Addresses issue number
#1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,26 @@
...
-You need to have Apache Cordova configured. Run `npm install` to install the Cordova CLI. Once you have the CLI installed, run `cordova run browser` in your terminal.
+Once you have the CLI installed you are ready to launch the app.
 
-References: [https://cordova.apache.org/](https://cordova.apache.org/)
+**For Browser Platform**
+Simply, run the command `cordova run browser` in your terminal. Then, open the browser and enter the url as shown in the terminal,i.e. `localhost:8080/index.html`
+Now, you can use the Notes-app.
+
+**For Android Platform** 
+You need to first setup your enviornment for running an android application. For that you need to download:
```
Comment (manaswinidas) [inline README.md:16]: typo: enviornment -> environment

File: README.md [hunk 2]
```diff
@@ -1,7 +1,26 @@
...
+1. [JDK 8](https://www.oracle.com/java/technologies/javase-jdk8-downloads.html)
+2. [Gradle](https://gradle.org/install/)
+3. [Android SDK](https://developer.android.com/studio/index.html)
+
+After installing all the listed applications, you need to [Set up your enviornment variable](https://cordova.apache.org/docs/en/latest/guide/platforms/android/index.html#setting-environment-variables) accordingly.
```
Comment (manaswinidas) [inline README.md:21]: typo: enviornment -> environment

File: README.md [hunk 3]
```diff
@@ -1,7 +1,26 @@
...
+Then, you are ready to run the Notes-app as an android application. To run an android app you either need to [Set up an emulator](https://cordova.apache.org/docs/en/latest/guide/platforms/android/index.html#setting-up-an-emulator) or to run on your phone you need to [Set up a hardware device](https://developer.android.com/studio/run/device).
+
+To run the app in android run the command `cordova run android` in your terminal.
+
+**For other platforms like OSX,Electron or if you need any other help in setting cordova CLI or your enviornment for the application you can look at the [Cordova Documentation site](https://cordova.apache.org/docs/en/latest/)**
```
Comment (manaswinidas) [inline README.md:26]: typo: enviornment -> environment

**Comment by Nikhil-Ladha** [comment]
This has always been my one of the issues😅. I will update it.

**Pull request closed without merging**